opendbc/car/lateral.py

```diff
@@ -120,9 +120,9 @@ def apply_steer_angle_limits_vm(apply_angle: float, apply_angle_last: float, v_e
   max_angle = get_max_angle_vm(v_ego_raw, VM, limits)
   new_apply_angle = np.clip(new_apply_angle, -max_angle, max_angle)
 
-  # angle is current angle when inactive
+  # angle is current angle when inactive, but still limited to max lat accel to prevent faults.
   if not lat_active:
-    new_apply_angle = steering_angle
+    new_apply_angle = np.clip(steering_angle, -max_angle, max_angle)
 
   # prevent fault
   return float(np.clip(new_apply_angle, -limits.ANGLE_LIMITS.STEER_ANGLE_MAX, limits.ANGLE_LIMITS.STEER_ANGLE_MAX))
```

Comment:
Option B is:

```suggestion
  return float(np.clip(new_apply_angle, -max_angle, max_angle))
```

Reply:
if the max angle is bigger than the actual limits, it would fault, so this is not good